app/modules/components/issues-table/issues-table.jade

```diff
@@ -51,7 +51,10 @@ section.issues-options(
 section.issues-table.basic-table(ng-class="{empty: !ctrl.issues.length}")
     div.row.title
         div.level-field(data-fieldname="type" ng-if="!ctrl.sprintIssues")
-            span {{"ISSUES.TABLE.COLUMNS.TYPE" | translate}}
+            span.wide {{"ISSUES.TABLE.COLUMNS.TYPE" | translate}}
```

Comment:
Why are you using two classes to display `flex` and `none` if you could rely on a single class and the media query and avoid code repetition?